store/json_test.go

```diff
@@ -129,6 +129,14 @@ func TestKeyValuePairsAreWrittenAndReadCorrectly(t *testing.T) {
 		t.Fatalf("Failed to create KeyValueStore %v\n", err)
 	}
 
+	defer os.Remove(testFileName)
+
+	// Dump empty store.
+	_, err = kvs.Dump()
+	if err == nil {
+		t.Fatal("Expected store to not exist")
```

Comment:
We expect the store to exist right ? why the `nil` check and a `fatal` error ?

Reply:
The store struct does exist, but the actual json representation on disk doesn't exist yet.